### PR TITLE
Revise questions 2.1, 2.2, and 2.3 for ease of understanding.

### PR DESCRIPTION
**Pull request by hober**
***
<a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/pull/90.html" title="Last updated on Aug 12, 2020, 4:31 PM UTC (472d3c8)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3ctag/security-questionnaire/90/90c4e42...472d3c8.html" title="Last updated on Aug 12, 2020, 4:31 PM UTC (472d3c8)">Diff</a>

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -90,47 +90,97 @@ we've prepared [a list of these questions in Markdown](https://raw.githubusercon
...
+
+See also
+
+* [[#data-minimization]]
 
 <h3 class=question id="personal-data">
-  How does this specification deal with personal information or
-  personally-identifiable information or information derived thereof?
+  How does this specification deal with personal information,
+  personally-identifiable information, or information derived from them?
```
[the remark this answers is not in the document]
Comment (hober) [reply index.bs]: Addressed in 1446026.

File: index.bs [hunk 2]
```diff
@@ -90,47 +90,97 @@ we've prepared [a list of these questions in Markdown](https://raw.githubusercon
...
-* A feature which uses biometric data (fingerprints or retina scans)
-    should refuse to expose the raw data to the web, instead using the raw
-    data only to unlock some origin-specific and ephemeral secret and
-    transmitting that secret instead.
-* Including a factor of user mediation should be considered, in order to
-    ensure that no data is exposed without a user’s explicit choice (and
-    hopefully understanding). One way to achieve this may be the use of
-    Permission API [[PERMISSIONS]], or additional dialogs like in Payment
-    Request API [[PAYMENT-REQUEST]]
+Personal information is data about a user
```
[the remark this answers is not in the document]
Comment (hober) [reply index.bs]: Addressed in 1446026.

File: index.bs [hunk 3]
```diff
@@ -90,47 +90,97 @@ we've prepared [a list of these questions in Markdown](https://raw.githubusercon
...
+(for example, their home address),
+or information that could be used to identify a user,
+such as an alias or email address.
+
+Note: Personal information is
+distinct from personally identifiable information
+(<abbr title="personally identifiable information">PII</abbr>).
+PII is a legal concept,
+the definition of which varies from jurisdiction to jurisdiction.
```
[the remark this answers is not in the document]
Comment (hober) [reply index.bs:134]: Addressed in 1446026.

**Comment by hober** [comment]
cc @samuelweiler, who also offered to review this

File: index.bs [hunk 4]
```diff
@@ -90,47 +90,102 @@ we've prepared [a list of these questions in Markdown](https://raw.githubusercon
   and for what purposes is that exposure necessary?
 </h3>
 
-Just because information can be exposed to the web doesn’t mean that it
-should be. How does exposing this information to an origin benefit a user?
-Is the benefit outweighed by the potential risks?  If so, how?
+Just because information **can** be exposed to the web
+doesn’t mean that it **should** be.
+Does your feature expose information to origins?
```
Comment (torgo) [inline index.bs:95]: I think this needs clarification.

Comment (hober) [reply index.bs:95]: I've revised this text, @torgo. What do you think?

Comment (torgo) [reply index.bs:95]: Rocks.

**Pull request merged**